dewy/documents/models.py

```diff
@@ -5,9 +5,11 @@
 
 
 class CreateRequest(BaseModel):
-    """The name of the collection the document should be added to."""
+    """The name of the collection the document should be added to. Either `collection` or `collection_id` must be provided"""
+    collection: Optional[str] = None
 
-    collection_id: int
+    """The id of the collection the document should be added to. Either `collection` or `collection_id` must be provided"""
```

Comment:
What happens if they're both provided and disagree? We should implement a `pydantic` validator to enforce the cosntraint that exactly one is specified (will also automaticalyl report the JSON errors, etc.).